xml/dm_registers.xml

```diff
@@ -508,15 +508,40 @@
 
     <!-- =============== system bus mastering =============== -->
 
+    <register name="System Bus Address 127:96" short="sbaddress3" address="0x37">
+        If \Fsbasize is less than 97, then this register is not present.
+
+        <field name="address" bits="31:0" access="R/W" reset="0">
+            Accesses bits 127:96 of the physical address in {\tt sbaddress} (if
+            the system address bus is that wide).
+        </field>
+    </register>
+
     <register name="System Bus Access Control and Status" short="sbcs" address="0x38">
-        <field name="0" bits="31:21" access="R" reset="0" />
-        <field name="sbsingleread" bits="20" access="W1" reset="0">
-            When a 1 is written here, triggers a read at the address in {\tt
-            sbaddress} using the access size set by \Fsbaccess.
+        <field name="version" bits="31:29" access="R" reset="1">
```

Comment:
shouldn't this be called `sbversion` to avoid alias issues with `\Fversion` macros (maybe not an actual compile issue because we're currently using `versionlo` and `versionhi`). All the other sb-related bits seem to be nicely prefixed with `sb`

Reply:
Also, why is it 3 bits? vs byte-aligned 4 bits?

Reply:
Yes, sbversion is better.

3 bits because how many versions do we really anticipate? I'd rather save the other bits for some future use. We can always widen the field later.